overlord/configstate/configcore/network_test.go

```diff
@@ -36,29 +36,24 @@ type networkSuite struct {
 	configcoreSuite
 
 	mockNetworkSysctlPath string
-	restores              []func()
 	mockSysctl            *testutil.MockCmd
 }
 
 var _ = Suite(&networkSuite{})
 
 func (s *networkSuite) SetUpTest(c *C) {
 	s.configcoreSuite.SetUpTest(c)
-	dirs.SetRootDir(c.MkDir())
-	s.restores = append(s.restores, release.MockOnClassic(false))
+	s.BaseTest.AddCleanup(release.MockOnClassic(false))
```

Comment:
Nice, thank you! Fwiw, s.AddCleanup() is fine here, no need to specifc s.BaseTest explicitly (or is there a good reason for this?).